lib/Standalone/ConvertTppToXsmm.cpp

```diff
@@ -47,11 +47,17 @@ struct ConvertTppMatmulOp : public OpRewritePattern<MatmulOp> {
     MemRefType memrefB = matmulOp.getMatrixBType();
     int64_t m = memrefC.getShape()[0];
     int64_t n = memrefC.getShape()[1];
-    int64_t k = memrefA.getShape()[1];
+    int64_t k = memrefB.getShape()[0];
```

Comment:
why this change?

Reply:
This is a stray change, I will revert this.